OpenRA.Mods.Common/Traits/Power/CanPowerDown.cs

```diff
@@ -20,9 +20,13 @@ public class CanPowerDownInfo : UpgradableTraitInfo, Requires<PowerInfo>
 		public readonly bool CancelWhenDisabled = false;
 
 		public readonly string IndicatorImage = "poweroff";
-		public readonly string IndicatorSequence = "offline";
+		[SequenceReference("IndicatorImage")] public readonly string IndicatorSequence = "offline";
 
-		public readonly string IndicatorPalette = "chrome";
+		[PaletteReference] public readonly string IndicatorPalette = "chrome";
+
+		public readonly string NotificationType = "Sounds";
```

Comment:
Not sure what you mean there. My request makes this *more* configurable than you currently have it, and making it match all the other traits helps with documentation.